libs/utils/utils.js

```diff
@@ -1052,6 +1052,10 @@ async function checkForPageMods() {
 
 async function loadPostLCP(config) {
   await decoratePlaceholders(document.body.querySelector('header'), config);
+  const sk = document.querySelector('helix-sidekick');
+  if (sk) {
+    import('./sidekick-decorate.js').then((mod) => { mod.default(sk); });
+  }
```

Comment:
Nit: You could do 1 line in here
```suggestion
  if (sk) import('./sidekick-decorate.js').then((mod) => { mod.default(sk); });
```